src/Button/Button.jsx

```diff
@@ -1,13 +1,42 @@
 import MaterialButton from '@material-ui/core/Button'
+import PropTypes from 'prop-types'
 import React from 'react'
 
-/**
- * Overrides the Material Button component with some custom styles.
- */
+/** The `<Button>` component represents a clickable button. */
```

Comment:
If we use the docgen syntax (e.g. `/** lorem ipsum */`), then the class comment and prop descriptions will be automatically included in the "Overview" section.